crates/wasi-nn/examples/classification-example/src/main.rs

```diff
@@ -1,36 +1,58 @@
+use image::{DynamicImage};
+use image::io::Reader;
 use std::convert::TryInto;
 use std::fs;
 use wasi_nn;
 
+// TODO change this to wasi_nn::GRAPH_ENCODING_TENSORFLOW once wasi-nn is updated.
+// const GRAPH_ENCODING_TENSORFLOW: u8 = 1;
```

Comment:
Looks like some updates are needed once a new version of the bindings crate is published... which is needed for CI to pass as well, I think.